single-line-diagram/single-line-diagram-core/src/main/java/com/powsybl/sld/layout/pathfinding/Point.java

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.sld.layout.pathfinding;
+
+import java.util.*;
+
+/**
+ * @author Thomas Adam <tadam at neverhack.com>
+ */
+public class Point {
+
+    private int x;
+    private int y;
+
+    Point(int x, int y) {
+        this.x = x;
+        this.y = y;
+    }
+
+    public int x() {
+        return x;
+    }
+
+    public int y() {
+        return y;
+    }
+
+    public Point setX(int x) {
+        this.x = x;
+        return this;
+    }
+
+    public Point setY(int y) {
+        this.y = y;
+        return this;
+    }
+
+    public int manhattanDistance(Point other) {
+        return Math.abs(this.x - other.x) + Math.abs(this.y - other.y);
+    }
+
+    public int angleBetween(Point a, Point b) {
```

Comment:
The name here is unclear to me. Do we really compute an angle or a scalar product?

Reply:
Method removed